src/service/chat.js

```diff
@@ -58,4 +59,43 @@ const sendSystemMessage = async (
   }
 };
 
-export {createChannel, sendSystemMessage};
+const sendAnonymousDMOtherProfile = async ({
+  user_id,
+  anon_user_info_color_code,
+  anon_user_info_color_name,
+  anon_user_info_emoji_code,
+  anon_user_info_emoji_name,
+  message
+}) => {
+  const payload = {
+    members: [user_id],
+    message,
+    anon_user_info_emoji_name,
+    anon_user_info_emoji_code,
+    anon_user_info_color_name,
+    anon_user_info_color_code
+  };
+
+  const response = await anonymousApi.post('/chat/init-chat-anonymous', payload);
+
+  if (response.status === 200) {
+    return Promise.resolve(response.data?.data);
+  }
+  return Promise.reject(response.data?.data);
+};
+
+const sendSignedDMOtherProfile = async ({user_id, message}) => {
+  const payload = {
+    members: [user_id],
+    message
+  };
+
+  const response = await anonymousApi.post('/chat/init-chat', payload);
```

Comment:
Mas @robyhuzwandar this doesn't use anonymousApi, use baseApi instead that uses signedToken